packages/tqdm/test_tqdm.py

```diff
@@ -0,0 +1,13 @@
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide(packages=["tqdm"], pytest_assert_rewrites=False)
+def test_tqdm(selenium):
+    import warnings
+
```

Comment:
We're getting there...
```suggestion
    import time
```